docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       MYSQL_ROOT_PASSWORD: somePassword
 
   sshproxy:
-    build: ./tests/env/sshproxy
+    build: ./tests/Old/env/sshproxy
```

Comment:
maybe we should keep the sshproxy setup in the main tests dir?

Reply:
I think we can, I just wanted to start without anything old and then move into it.

E.g. as you mentioned in #88 create ssh tests using datadirs